js/script.js

```diff
@@ -0,0 +1,154 @@
+                              <animateTransform attributeName="transform" type="rotate" calcMode="linear" values="0 50 50;-360 50 50" keyTimes="0;1" dur="1s" begin="0s" repeatCount="indefinite"></animateTransform>
+                          </circle>
+                        </svg>
+                    <figure>
+                  `;
+        break;
+      case "success":
+        message = `
+                    <div class="success-box">
+                        <svg class="checkmark checkmark-success" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 52 52">
+                            <circle class="checkmark-circle" cx="26" cy="26" r="25" fill="none"/>
+                                <path class="checkmark-check" fill="none" d="M14.1 27.2l7.1 7.2 16.7-16.8"/>
+                        </svg>
+                        <strong>Sua mensagem foi enviada!</strong>
+                        <div class="column is-narrow has-text-centered">
+                            <button type="button" title="Retornar" class="button is-info back-button">Retornar</button>
+                        </div>
+                    </div>
+                  `;
+        break;
+      case "fail":
+        message = `
+                    <div class="error-box">
+                        <svg class="checkmark  checkmark-error" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 52 52">
+                            <circle class="checkmark-circle" cx="26" cy="26" r="25" fill="none"/>
+                            <path class="checkmark-check" fill="none" d="M16 16 36 36 M36 16 16 36"/>
+                        </svg>
+                        <strong>Ocorreu um erro :( <br/> Tente novamente mais tarde</strong>
+                        <div class="column is-narrow has-text-centered">
+                            <button type="button" title="Retornar" class="button is-info back-button">Retornar</button>
+                        </div>
+                    </div>
+                  `;
+        break;
+      // no default
+    }
+    ui.container.insertAdjacentHTML(
+      "beforeend",
+      `
+        <div class="message-alert">
+            <div class="is-${status}">
+                    ${message}
+            </div>
+        </div>
+    `,
+    );
+  };
+
+  const returnButtonAction = () => {
+    const messagesAlert = document.querySelectorAll(".message-alert");
+
+    ui.form.style.display = "block";
+
+    ui.fields.forEach((field) => {
+      field.value = "";
+    });
+
+    messagesAlert.forEach((message) => {
+      message.remove();
+    });
+  };
+
+  const returnToForm = () => {
+    document.querySelectorAll(".is-loading").forEach((loadBox) => {
+      loadBox.style.display = "none";
+    });
+
+    const backButton = document.querySelector(".back-button");
+    backButton.addEventListener("click", returnButtonAction);
+  };
+
+  const submitMessage = (e) => {
+    e.preventDefault();
```

Comment:
why not destruct preventDefault if you are not using anything else from e variable? Or the variable name should be more specific, what e means? event? exception?